.github/scripts/topology_census.py

```
@@ -68,47 +68,60 @@ def _run_git(root: Path, *args: str) -> str:
 def _git_path_is_ignored(root: Path, relpath: str) -> bool:
     if not _git_repo_available(root):
         return False
-    result = subprocess.run(
-        ["git", "-C", str(root), "check-ignore", relpath],
-        capture_output=True,
-        text=True,
-        encoding="utf-8",
-        errors="replace",
-    )
+    try:
+        result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)